extensions/smallrye-health/deployment/src/main/java/io/quarkus/smallrye/health/deployment/SmallRyeHealthProcessor.java

```diff
@@ -52,10 +60,22 @@
     @ConfigRoot(name = "smallrye-health")
     static final class SmallRyeHealthConfig {
         /**
-         * The path of the health-checking servlet.
+         * The path of the legacy health-checking servlet.
          */
         @ConfigItem(defaultValue = "/health")
-        String path;
+        String healthPath;
+
+        /**
+         * The relative path of the liveness health-checking servlet.
+         */
+        @ConfigItem(defaultValue = "/live")
+        String livenessRelPath;
```

Comment:
I would drop the `Rel` part. Otherwise, you end up with: `quarkus.smallrye-health.liveness-rel-path`. Not very appealing.

If the other is `rootPath`, I think it's clear enough.